packages/app/src/version-service.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * @license
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export class VersionService {
```

Comment:
Feel like we can just use a plain Object instead of a class to save an additional file?

Reply:
Done.

Reply:
TBH classes have [very little overhead](http://www.typescriptlang.org/play/?target=1#code/MYGwhgzhAEBqCmAnCBLA9gOwMpIG4uHmgG8BYAKGmmEwgBdEBXYOtRACkXjABNMQAntBAoARojCIBALmj1EKDAHMANNC69+Q3ElSZZ8xUoCUJAL4UzQA), but if we're going to use a plain object, could we also export and use a function like this:

```typescript
function makeVersionService(library: string, service: string) {
  return {library, service};
}
```

It would make things more readable and would be useful in case we want to change the fields later or something?

Reply:
We also type this object with an interface, so it's still type safe.

I remember classes gave us treeshaking problems before (https://github.com/firebase/firebase-js-sdk/pull/2006), so I'm a little cautious of using them, but the output you linked doesn't have `Object.create` which was the culprit. Maybe `Object.create` is being avoided now, or it is only emitted under certain condition?